ai/Algorithm/auto_play.py

```diff
@@ -113,25 +114,24 @@ def _normal_start(self):
             InternalRefereeCommand.PREPARE_PENALTY_US: SimpleAutoPlayState.OFFENSE_PENALTY,
             InternalRefereeCommand.PREPARE_PENALTY_THEM: SimpleAutoPlayState.DEFENSE_PENALTY,
             RefereeCommand.NORMAL_START: self._analyse_game()
-        }.get(self.last_ref_command, RefereeCommand.NORMAL_START)
+        }.get(self.last_ref_state, RefereeCommand.NORMAL_START)
 
-    def _select_next_state(self):
-        referee = GameState().referee
+    def _select_next_state(self, ref_state: RefereeState):
         next_state = self.current_state
         # On command change
-        if self.last_ref_command != referee.command:
+        if self.last_ref_state != ref_state.command:
             next_state = {
-                RefereeCommand.HALT: SimpleAutoPlayState.HALT,
+                InternalRefereeCommand.HALT: SimpleAutoPlayState.HALT,
```

Comment:
whats the difference with internal command?

Reply:
RefereeCommand -> BLABLA_YELLOW, BLABLA_BLUE
InternalRefereeCommand -> BLABLA_US, BLABLA_THEM

Reply:
RefereeState s'occupe de la conversion, mais c'est vrai que l'on pourrait avoir un constreur de InternalRefereeCommand qui prend en entrée une RefereeCommand et qu'il le convertie.
Présentement, RefereCommand est uniquement utilisé par RefereeState.